sdk/ml/azure-ai-ml/tests/data_import/e2etests/test_data_import.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from devtools_testutils import AzureRecordedTestCase
+
+from azure.ai.ml import MLClient, load_data_import
+from azure.ai.ml.entities import PipelineJob, DataImport
+from azure.ai.ml.entities._inputs_outputs import Output
+from azure.ai.ml.entities._inputs_outputs.external_data import Database
+from azure.ai.ml.entities._job.data_transfer.data_transfer_job import DataTransferImportJob
+
+
+@pytest.mark.usefixtures(
+    "recorded_test",
+    "mock_code_hash",
+    "mock_asset_name",
+    "mock_component_hash",
+    "enable_environment_id_arm_expansion",
+)
+@pytest.mark.timeout(600)
+@pytest.mark.e2etest
+@pytest.mark.usefixtures("recorded_test", "mock_code_hash")
+@pytest.mark.data_experiences_test
+class TestDataImport(AzureRecordedTestCase):
+    # Please set ML_TENANT_ID in your environment variables when recording this test.
+    # It will to help sanitize RequestBody.Studio.endpoint for job creation request.
+    def test_data_import(self, client: MLClient) -> None:
```

Comment:
maybe need add recording files for e2e test

Reply:
Added successfully and related build checks passed